lib/ex_unit/test/runner_test.exs

```diff
@@ -0,0 +1,23 @@
+Code.require_file "test_helper.exs", __DIR__
+
+defmodule ExUnitTest do
```

Comment:
Let's change this to `ExUnit.RunnerTest` to mimic the module we are testing? And we should move the file to `test/ex_unit/runner_test.exs`. :)